src/components/steps/BasicSteps.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <el-steps :active="active" finish-status="finish" :simple=isSimple :direction="direction">
```

Comment:
is it possible to change the `finish-status` to "success" ? since it will be nice to have the completed steps as check marked ✔️